crates/sui-e2e-tests/tests/unwrapped_then_deleted_tests.rs

```diff
@@ -19,6 +19,8 @@ mod sim_only_tests {
     async fn test_no_more_dependency_on_wrapped_tombstone() {
         let mut _guard = ProtocolConfig::apply_overrides_for_testing(|_, mut config| {
             config.set_simplified_unwrap_then_delete(false);
+            // Given that we will never enable effect V2 before turning on simplified_unwrap_then_delete, we also need to disable effect V2 here.
```

Comment:
A better way to force this consistent is that in the implementation of `set_simplified_unwrap_then_delete`, set effects v2 to false.

Reply:
Sorry that I missed this comment. I'll fix it right way.